--- a/src/Mailjet/Client.php
+++ b/src/Mailjet/Client.php
@@ -74,7 +74,7 @@ class Client
      * @param string|null $secret   Mailjet API Secret
      * @param bool        $call     performs the call or not
      * @param array       $settings
      */
-    public function __construct(string $key, string $secret = null, bool $call = true, array $settings = [])
+    public function __construct(string $key, ?string $secret = null, bool $call = true, array $settings = [])
     {
         $this->setAuthentication($key, $secret, $call, $settings);
     }
